packages/nimble-components/src/select/specs/filerable-select-hld.md

```diff
@@ -0,0 +1,78 @@
+# Filterable Select
+
+## Problem Statement
+
+Our clients have a need of a filterable dropdown component that does not require arbitrary text as a value, but only values available in the dropdown. The filtering will have the following behaviors:
+
+-   The filter text will match _any_ text within the `textContent` of each `ListOption`
+-   Each time the dropdown is opened the filter text is cleared
+-   While the dropdown is opened all keystrokes except `<ArrowUp>` and `<ArrowDown>` will apply to the filter.
```

Comment:
The current implementation of the select uses `Home` and `End` to jump to the first or last item in the select. Will those now apply to the filter, or should they be called out here as keystrokes that won't apply to the filter?

Reply:
Called those keys out as not applying to the filter input.